packages/marketplace-ui/src/components/admin/LaunchSettingsModal.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  Text,
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalOverlay,
+  ModalHeader,
+  Alert,
+} from "@chakra-ui/react";
+import React from 'react';
+import { PublicKey } from '@solana/web3.js';
+import { DisburseFunds } from "..";
+import { useWallet } from "@solana/wallet-adapter-react";
+import { useTokenSwapFromId } from "@strata-foundation/react";
+import { useIsBountyAdmin } from "../../hooks/useIsBountyAdmin";
+  
+  
+interface LaunchSettingsModalProps {
+  id: PublicKey;
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+export const LaunchSettingsModal = ({ id, isOpen, onClose }: LaunchSettingsModalProps) => {
```

Comment:
Shouldn't be needed, lets just send them to the docs on the launch.

Reply:
@ChewingGlass I had thought this page was going replace the docs page and streamline the interactions to just here. 

When someone creates a new token where are they redirected to at the end of the creation process. The docs or this admin page? That will answer if we need this.

Reply:
This is the default path for *tokens*, for a launch you still go to docs. As that's effectively the launch admin page.

Reply:
Ah gotcha

Reply:
Done, redirects to docs instead